bubblewrap.c

```diff
@@ -1458,6 +1485,9 @@ main (int argc,
   /* We want sigchild in the child */
   unblock_sigchild ();
 
+  if (label_exec(opt_exec_label) == -1)
```

Comment:
I saw that and was not sure.  We could change this to setcon() instead of secexeccon() which will change the context of the bubblewrap tool to pid1.